stories/Guidelines.stories.mdx

```diff
@@ -0,0 +1,43 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta
+  title="Getting Started/Guidelines"
+  parameters={{
+    viewMode: 'docs',
+    previewTabs: { canvas: { hidden: true } }
+  }}
+/>
+
+# Developer Guidelines
+
+
+Various guidelines to ensure consistency and quality across the Upfluence Frontend Team.
+
+
```

Comment:
```suggestion
```

Reply:
Writes a Guideline, doesn't follow it in the Guidelines doc. 😂

Reply:
Do what I say, not what I do :grin:

Reply:
Exactly hahah